packages/modules/imodel-browser/src/containers/ITwinGrid/useITwinData.ts

```diff
@@ -22,6 +22,8 @@ export interface ProjectDataHookOptions {
   filterOptions?: ITwinFilterOptions;
   shouldRefetchFavorites?: boolean;
   resetShouldRefetchFavorites?: () => void;
+  isStaleData?: boolean;
```

Comment:
Comments should be added here to clarify as these are non-obvious.

To me, it seems like this would be better handled by having the hook return a function to refresh the data. It seems overcomplicated to have two input properties. You set `isStaleData`, which triggers an effect that calls `resetStaleData`, which changes `isStaleData`, which triggers the effect again.

Reply:
indeed that was a convoluted way to achieve the result. I updated the code to follow your suggestion, thanks!